.vscode/launch.json

```diff
@@ -89,6 +89,14 @@
             ],
             "cwd": "${workspaceFolder}/tests"
         },
+        {
+            "type": "lldb",
+            "request": "launch",
+            "name": "LLDB Node",
+            "program": "node",
+            "args": ["--expose_gc"],
+            "preLaunchTask": "Build Node Tests"
+        },
```

Comment:
What does this actually do? It doesn't seem to specify the file to run?

Reply:
It opens a node REPL, running in debug mode so you can add breakpoints etc. I found it pretty useful when developing for quickly testing things out (using `realm = require('.')`). Maybe I should rename it to `LLDB Node REPL`?